app/main/views/sign_out.py

```diff
@@ -26,11 +26,19 @@ def _sign_out_at_login_dot_gov():
 @main.route("/sign-out", methods=(["GET", "POST"]))
 def sign_out():
     # An AnonymousUser does not have an id
-    current_app.logger.info("HIT THE REGULAR SIGN OUT")
+
     if current_user.is_authenticated:
         # TODO This doesn't work yet, due to problems above.
```

Comment:
I know this comment isn't a part of this code change, but is it now still relevant to keep given what is being changed?

Reply:
It has not been super helpful to date, so I removed it.